src/test/resources/database/database.sql

```diff
@@ -6,7 +6,8 @@ CREATE TABLE Deposit (
     depositId CHAR(36) NOT NULL PRIMARY KEY,
     bagName VARCHAR(10000),
     creationTimestamp TIMESTAMP WITH TIME ZONE NOT NULL,
-    depositorId VARCHAR(64) NOT NULL
+    depositorId VARCHAR(64) NOT NULL,
+    origin VARCHAR(6) NOT NULL,
```

Comment:
Is this trailing comma allowed in SQL? I'm fairly sure it isn't in PostgreSQL.

```suggestion
    origin VARCHAR(6) NOT NULL
```

Reply:
If not allowed, then why do the tests succeed?